.github/workflows/ruby.yml

```diff
@@ -0,0 +1,68 @@
+name: Ruby
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
```

Comment:
what's the reasoning for this? thought we'd want CI to run on every push to the repo?